Zeromq doesn't build on Electron 9

### Prerequisites

* [x] Have you checked the solutions in our [troubleshooting guide](https://nteract.gitbooks.io/hydrogen/docs/Troubleshooting.html) for common problems ?
* [x] Have you checked that your issue [isn't already filed](https://github.com/nteract/hydrogen/issues) ?

### Description

Hydrogen fails to build on Atom with Electron 9. 

Full log is included in the log section. This is a summary:
```
❯ apm-dev rebuild --verbose
Rebuilding modules failed
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\MSBuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\aminy\Documents\GitHub\JavaScript\@atom-ide-community\atom\out\Atom Dev x64\resources\app\apm\node_modules\npm\node_modules\node-gyp\lib\build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:200:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Windows_NT 10.0.19042
gyp ERR! command "C:\\Users\\aminy\\Documents\\GitHub\\JavaScript\\@atom-ide-community\\atom\\out\\Atom Dev x64\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\aminy\\Documents\\GitHub\\JavaScript\\@atom-ide-community\\atom\\out\\Atom Dev x64\\resources\\app\\apm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\.atom\packages\Hydrogen\node_modules\zeromq
gyp ERR! node -v v12.4.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zeromq@5.2.0 install: `node scripts/prebuild-install.js || (node scripts/preinstall.js && node-gyp rebuild)`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the zeromq@5.2.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\aminy\.atom\.apm\_logs\2020-12-16T23_12_45_444Z-debug.log
```

### Steps to Reproduce

1. Build Atom in https://github.com/atom/atom/pull/21805
2. Try to build Hydrogen with the included apm under `./out/Atom Dev x64/resources/app/apm/bin/apm`


### Versions

<!-- You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running. -->

OS version:
[Your OS's version]

> `atom --version`
<!-- Paste the the output of `atom --version` in the backticks below -->
```
❯ atom-dev9 --version

Atom    : 1.56.0-dev-ffaba34b3
Electron: 9.3.5
Chrome  : 83.0.4103.122
Node    : 12.14.1
```

> `apm --version`
<!-- Paste the the output of `apm --version` in the backticks below -->
```
❯ apm-dev --version
apm  2.5.2
npm  6.14.8
node 12.4.0 x64
atom 1.56.0-dev-ffaba34b3
python 3.6.8
git 2.29.2.windows.3
visual studio
```

Hydrogen version:
<!-- You can check this information in the settings view or via running `apm show hydrogen` -->
Hydrogen 2.14.6

### Plugins

Have you installed and activated any of the Hydrogen plugins below ?
- [ ] hydrogen-python
- [ ] Hydrogen Launcher
- [ ] Data Explorer

### Logs

Here is a full log:
[2020-12-16T23_12_45_444Z-debug.log](https://github.com/nteract/hydrogen/files/5705979/2020-12-16T23_12_45_444Z-debug.log)


### Expected behavior

[A clear and concise description of what you expected to happen.]

### Screenshots

[If applicable, add screenshots to help explain your problem.]

### Additional context

[Add any other context about the problem here.]

--- a/package.json
+++ b/package.json
@@ -69,7 +69,7 @@
     "atom-select-list": "^0.7.0",
     "escape-carriage": "^1.2.0",
     "escape-string-regexp": "^4.0.0",
-    "jmp": "^2.0.0",
+    "jmp": "https://github.com/aminya/jmp",
     "kernelspecs": "^2.0.0",
     "lodash": "^4.14.0",
     "mathjax-electron": "^3.0.0",

